massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -675,11 +681,14 @@ impl ConsensusWorker {
                     {}
                 );
                 if response_tx
-                    .send(
-                        self.block_db
-                            .get_active_block(&block_id)
-                            .map(|v| v.block.clone()),
-                    )
+                    .send(self.block_db.get_active_block(&block_id).and_then(|v| {
```

Comment:
I'm going to remove the `get_active_block` command, because it seems to only be used in tests and isn't necessary anymore with the use of storage.